src/locale/locales/es/messages.po

```diff
@@ -73,112 +73,112 @@ msgstr ""
 #: src/components/ProfileHoverCard/index.web.tsx:398
 #: src/screens/Profile/Header/Metrics.tsx:23
 msgid "{0, plural, one {follower} other {followers}}"
-msgstr ""
+msgstr "{0, plural, un {follower} unos {followers}}"
 
 #: src/components/ProfileHoverCard/index.web.tsx:402
 #: src/screens/Profile/Header/Metrics.tsx:27
 msgid "{0, plural, one {following} other {following}}"
-msgstr ""
+msgstr "{0, plural, un {following} unos {following}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:312
 msgid "{0, plural, one {Like (# like)} other {Like (# likes)}}"
-msgstr ""
+msgstr "{0, plural, un {Like (# like)} unos {Like (# likes)}}"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:434
 msgid "{0, plural, one {like} other {likes}}"
-msgstr ""
+msgstr "{0, plural, un {like} unos {likes}}"
 
 #: src/components/FeedCard.tsx:213
 #: src/view/com/feeds/FeedSourceCard.tsx:300
 msgid "{0, plural, one {Liked by # user} other {Liked by # users}}"
-msgstr ""
+msgstr "{0, plural, un {Liked by # user} unos {Liked by # users}}"
 
 #: src/screens/Profile/Header/Metrics.tsx:59
 msgid "{0, plural, one {post} other {posts}}"
-msgstr ""
+msgstr "{0, plural, un {post} unos {posts}}"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:418
 msgid "{0, plural, one {quote} other {quotes}}"
-msgstr ""
+msgstr "{0, plural, una {quote} unas {quotes}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:269
 msgid "{0, plural, one {Reply (# reply)} other {Reply (# replies)}}"
-msgstr ""
+msgstr "{0, plural, una {Reply (# reply)} unas {Reply (# replies)}}"
 
 #: src/view/com/post-thread/PostThreadItem.tsx:400
 msgid "{0, plural, one {repost} other {reposts}}"
-msgstr ""
+msgstr "{0, plural, un {repost} unos {reposts}}"
 
 #: src/view/com/util/post-ctrls/PostCtrls.tsx:308
 msgid "{0, plural, one {Unlike (# like)} other {Unlike (# likes)}}"
-msgstr ""
+msgstr "{0, plural, un {Unlike (# like)} unos {Unlike (# likes)}}"
```

Comment:
```suggestion
msgstr "{0, plural, one {Unlike (# me gusta)} other {Unlike (# me gusta)}}"
```
I have no idea how `Unlike` (in the sense of removing a like from a post) should be translated here

Reply:
`Unlike like` should be `No me gusta` , plurals should be `No me gustan`

Reply:
meaby
`msgstr "{0, plural, one {No (# me gusta)} other {No (# me gustan)}}"`

but.... dont know the context

Reply:
This string is an `accessibilityLabel` for the like button when someone has already liked a post. So in that situation, say the post has 5 likes, the person's screen reader would read out in English: "Unlike, 5 likes".

This lets the person using the screen reader know what the button does (it unlikes the post or in other words it removes the like from the post) and how many likes the post currently has (5).

https://github.com/bluesky-social/social-app/blob/090ac041c30f4ef887e6a2b17aa254d029cba9ce/src/view/com/util/post-ctrls/PostCtrls.tsx#L305-L310

Reply:
then your latest suggestion should be ok
`msgstr "{0, plural, one {No me gusta (# me gusta)} other {No me gusta (# me gusta)}}"`
we need to check the others issues with "likes" translations

Reply:
ok, fixed